src/ansys/dpf/post/harmonic_mechanical_simulation.py

```diff
@@ -97,6 +99,13 @@ def _get_result(
                 List of IDs of elements to get results for.
             named_selections:
                 Named selection or list of named selections to get results for.
+            expand_cyclic:
+                For cyclic problems, whether to expand the sectors.
```

Comment:
Does sectors start at 1 or 0

Reply:
It is now forced it to 1-based indexing, reflecting what is shown in the label index when doing no expansion (it shows sector 1).
Stages however are shown as starting at 0.